src/renderer/modals/VoteTron/index.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import React, { PureComponent } from "react";
+import Modal from "~/renderer/components/Modal";
+import Body from "./Body";
+import type { StepId } from "./types";
+type State = {
+  stepId: StepId,
+};
+
+const INITIAL_STATE = {
+  stepId: "castVotes",
+};
+
+class VoteModal extends PureComponent<{ name: string }, State> {
```

Comment:
Why is this a class component?

Reply:
same copy pasted the same way of doing stepper modals we do elsewhere. :sweat_smile: